doc/lightning-commando-listrunes.7.md

```diff
@@ -0,0 +1,60 @@
+lightning-commando-listrunes -- Command to List Stored Runes
+===================================================================
+
+SYNOPSIS
+--------
+
+**commando-listrunes**
+
+DESCRIPTION
+-----------
+
+The **commando-listrunes** RPC command returns a list of runes stored in the datastore.
+
+EXAMPLES
+--------
+
+This lists all runes saved in the datastore:
+
+    $ lightning-cli commando-listrunes
+   {
+      "runes": [
+         {
+            "rune": "26s8Ef4kEylq1z0CLfTLMIl5ddyxwLpgsy5qJWWv-EA9NA=="
+         },
+         {
+            "rune": "iI3RdCKkX4TOe-63_k7EGEy8Hg-CY8Mdrtrdd0slAcQ9NQ=="
+         }
+      ]
+   }
+
+RETURN VALUE
+------------
+
+[comment]: # (GENERATE-FROM-SCHEMA-START)
+On success, an object is returned, containing:
+
+- **runes** (array): the array of objects
+
+[comment]: # (GENERATE-FROM-SCHEMA-END)
```

Comment:
This is definitely wrong, since the tooling will try to regenerate this from the schema, which does not yet exist!  Either document the complete thing at the end, *or* don't use the GENERATE-FROM-SCHEMA-START here, but maybe `[comment]: # (FIXME: generate from schema)` instead?